commands/livecmd.go

```diff
@@ -45,13 +46,15 @@ func GetLiveCommand(ctx context.Context, _, version string) *cobra.Command {
 	}
 
 	f := newFactory(liveCmd, version)
+	invFactory := live.NewClusterClientFactory()
```

Comment:
I think we can just keep this inside the cmdstatus package?

Reply:
@Morten Do you mean the NewClusterClientFactory function and the struct related to it or you are just saying this line could be placed inside the cmdstatus?

Reply:
I think the test utilizes the FakeClusterClient which is a different one, so the invFactory needs to be an input argument, otherwise the test will fail. But I think it's ok to move the implementation of the client factory into cmdstatus.